### PR TITLE
[CMLG-025] Change all font to Helvetica

### PR DESCRIPTION
**Pull request by EileenSiqi**
Issue:
Client wants all the font to be Helvetica, but some local defined font currently are not.

Solution:
Clear local defined font.

Risk:
Change in headerbar component and aboutus component.

Reviewed by:
Emily, Yujia, Kevin, Annie, Dave

### CODE REVIEW

File: src/components/HeaderBar.js [hunk 1]
```diff
@@ -65,17 +65,17 @@ class HeaderBar extends React.Component {
             <div>
                 <NavLink activeStyle={{ textShadow: "2px 2px 5px #5DADE2" }} exact to="/">
                     <li>
-                        <i className={ ` fas fa-home ${ this.props.darkMode ? "dark-mode-icon" : "" } ` }>&nbsp;Home</i>
+                        <i className={ ` fas fa-home ${ this.props.darkMode ? "dark-mode-icon" : "" } ` }><span className="fa-icon-inner-text">&nbsp;Home</span></i>
```
Comment (annithinggoes) [inline src/components/HeaderBar.js:68]: is this actually a fontawesome text or something else? dont need to have the fa-icon in the classname imo

**Comment by annithinggoes** [comment]
image
Do we want the font to be bold or normal weighting? 
I changed it by, though once again its this changing a predefined class, not sure how others feel about it.
image

**Comment by zyan225** [comment]
> 
> 
> image
> image
> Do we want the font to be bold or normal weighting?
> I changed it by
> image
> Though once again its this changing a predefined class, not sure how others feel about it.

I think normal is good. If we are changing the pre-defined class, can we use currentFileName.modules.css as the file name? And then use `import styles from '......'` and styles.className instead? So we make sure the changes are only applied in this file.

File: src/components/HeaderBar.js (hunk 1 above)
Comment (EileenSiqi) [reply src/components/HeaderBar.js]: I have updated the class name xD

**Comment by EileenSiqi** [comment]
So we stick with the default font-weight, right?

**Comment by zyan225** [comment]
> 
> 
> So we stick with the default font-weight, right?

Yes, just the normal (unbolded) one is fine.

**Comment by EileenSiqi** [comment]
> > So we stick with the default font-weight, right?
> 
> Yes, just the normal (unbolded) one is fine.

Oh, I misunderstood, just updated.

**Review by zyan225 — APPROVED** [review]

**Review by annithinggoes — APPROVED** [review]

**Pull request merged**